### DISCUSSION TITLE
Port model from Power BI to SSAS Tabular

### DISCUSSION THREAD

**Discussion by thetom88** [TabularEditor/TabularEditor3#1334] [🙋♀️TE3 Q&A] [answered]
I've developed a model in Power BI Desktop(compatibility level 1567) and I'm now unfortunately being asked to port my solution to SQL Server Analysis Services 2017(Tabular instance, compatibility level 1400). Is there a way to "downgrade" the compatibility level of my model using TE3? 
I made an attempt using TE2 by copying the tables manually, but, as expected, it failed.

**Comment by MariaJoseFF** [comment, marked as answer]
Hi @thetom88, 

We don't support compatibility downgrades in Tabular Editor. Our recommendation is to change the Compatibility Mode to "AnalysisServices" and the Compatibility Level to "1400" in the Database section of the model properties in Tabular Editor, then try to deploy the model to SQL Server Analysis Services and change the uncompatible components according to the error messages you will get.

I hope this helps; otherwise, if you need further assistance, we would need the error messages from SSAS to know which objects you need to change.

BR Maria Jose
Tabular Editor 3

**Comment by thetom88** [comment]
Hello @mjf-TabularEditor ,
Thank you for your answer. Based on your considerations, I guess the results won't be much different from my first tentative with TE2. I'll continue with the manual migration, I'll consider it as a chance to review my solution and maybe improve something.
Thank you again for your answer.
